addons/mail/controllers/discuss.py

```diff
@@ -551,6 +551,11 @@ def audio_worklet_processor(self):
             ]
         )
 
+    @http.route('/mail/thread/data', methods=['POST'], type='json', auth='user')
+    def mail_thread_attachment(self, thread_model, thread_id, **kwargs):
```

Comment:
forgot to rename the method following change

Reply:
also need to put the route in the same position in the file as in master

Reply:
also should have the same signature and return value

Reply:
Done !

Reply:
The route still doesn't take the request_list param and attachments return key.

Reply:
Yes, because it trigger a "var declared by unused" error on Runbot. I removed it because it's not used and didn't want to use it just because.

Reply:
But it needs to be used in a condition same as in master, we don't want the code to be any different.